### PR TITLE
Updated to jupyterlab 2.2.10

### PR DESCRIPTION
**Pull request by aniedzielaAnaconda**
Updated to version 2.2.10

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ source:
 build:
   noarch: python
   number: 0
+  skip: True # [s390x]
   script: {{ PYTHON }} -m pip install --install-option="--skip-npm" . -vv
```
Comment (varlackc) [inline recipe/meta.yaml:16]: Currently the package is failing for the `ppc64le` architecture
We might need to add a `skip` option for the `ppc64le` and add a comment explaining that we are not supporting that architecture at this point.

Comment (aniedzielaAnaconda) [reply recipe/meta.yaml:16]: Thank you @varlackc for the feedback. A bit strange that the `ppc64le` has failed, as it was not failing before. I will double check with @katietz before.

Comment (aniedzielaAnaconda) [reply recipe/meta.yaml:16]: The problem is caused by Prefect. Will close and reopen PR to restart the builds.

**Comment by aniedzielaAnaconda** [comment]
Close PR, will reopen in in few minutes.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,6 +12,7 @@ source:
 build:
   noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: If we will build an **arch-specific** package then `noarch: python` should be removed.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@ source:
...
   number: 0
+  skip: True # [s390x]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:15]: Also skip `py<35` if we'll remove `noarch python`

**Review by skupr-anaconda — COMMENTED** [review]
We should use a plain `python` **without** pinning for an arch-specific package

**Pull request merged**